### ISSUE TITLE
Hello Causes Screen Flickering

### ISSUE THREAD

**Issue by Kandelai** [n4n0GH/hello#69]
Using Hello on my rig, i5-9600k/Nvidia GTX 1660 Super using proprietary Nvidia drivers on Manjaro KDE stable. Everything worked well, until the screen flickering started. I added Triple Buffer to the Nvidia config, all the typical flickering fixes, etc, but it would still happen. It usually occurred if I started a gtk app or mostly and always if I exited a video game. Other times it was just random, a video playing would set it off etc. I would have to log out, and log back in again to get it to stop until something triggered it again.

I uninstalled Hello and the theme from my system, and all is perfect now, no screen flickering.

**Comment by n4n0GH** [comment]
Can you check if the solutions posted in #19 would fix your issue? It's sadly nothing that I can work on with the window decorations alone, as it appears to be a driver/gpu related issue.

**Comment by Kandelai** [comment]
It is definitely in the Hello shaders install found in Desktop Effects. I am using the Hello theme, but turned off the Hello shaders, and no flickering. 

I read #19 and it sounds exactly the same issue. I will try the fix which I guess is at the very end, and let you know.

**Comment by Kandelai** [comment]
Follow up. I got it wrong above. The issue for me is not with the Hello Shader, but it happens when applying the theme, Windows Decorations and Plasma Style. I can select another theme and run hello shader and it works without flickering coming out of a game, or whatever. 

I did try the fix in #19 , but that fix is for an Intel GPU. I have Nvidia, and it didn't work regardless. Just putting this here for more context.
